### PR TITLE
Added a Card demo

### PR DESCRIPTION
**Pull request by HansMuller**
Just a simple clone of the "Travel stream" screenshot that appears in https://www.google.com/design/spec/components/cards.html

### CODE REVIEW

File: examples/material_gallery/lib/demo/cards_demo.dart [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                padding: const EdgeDims.all(16.0),
+                child: new Column(
+                  justifyContent: FlexJustifyContent.start,
+                  alignItems: FlexAlignItems.start,
+                  children: <Widget>[
+                    // three line description
+                    new Text(destination.description[0], style: descriptionStyle),
+                    new Text(destination.description[1], style: descriptionStyle),
+                    new Text(destination.description[2], style: descriptionStyle),
+                    // share,explore buttons
```
Comment (Hixie) [inline examples/material_gallery/lib/demo/cards_demo.dart]: typo (missing space)

File: examples/material_gallery/lib/demo/cards_demo.dart [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    description: const <String> [
+      'Number 10',
+      'Whitehaven Beach',
+      'Whitsunday Island, Whitsunday Islands'
+    ]
+  ),
+  const TravelDestination(
+    assetName: 'packages/flutter_gallery_assets/kangaroo_valley_safari.png',
+    title: 'Kangaroo Valley Safari',
+    description: const <String> [
```
Comment (Hixie) [inline examples/material_gallery/lib/demo/cards_demo.dart]: no space between type and array literal brackets

**Comment by Hixie** [comment]
LGTM.

How standard are these card layouts? Should we have a card.dart with a bunch of predefined nice-looking layouts?

**Comment by HansMuller** [comment]
Absolutely.  There are quite a few Card features still TBD, including supporting the standard layouts.

**Pull request merged**